storage/rocksdb/rdb_threads.cc

```diff
@@ -61,17 +64,6 @@ int Rdb_thread::create_thread(const std::string &thread_name
   int err = mysql_thread_create(background_psi_thread_key, &m_handle, nullptr,
                                 thread_func, this);
 
-  if (!err) {
```

Comment:
Logged here https://blueprints.launchpad.net/percona-server/+spec/5.7-myrocks-add-setname and here https://jira.percona.com/browse/MYR-93.